### PR TITLE
Table borders for LEGEND.MD

### PR DESCRIPTION
**Pull request by AkalUstat**

### CODE REVIEW

File: LEGEND.md [hunk 1]
```diff
@@ -24,31 +24,33 @@
...
+| cmd + 1                                    | Waheguru Slide                              |
+| cmd + 2                                    | Mool Mantra Slide                           |
+| cmd + 3                                    | Empty Slide                                 |
+| cmd + 4                                    | Anand Sahib (Bhog)                          |
+| cmd + 5                                    | Help Guide                                  |
+| cmd + 6                                    | Shortcut Legend                             |
+| cmd + /                                    | Jump to search bar                          |
+| cmd + g                                    | Add Dhan Guru slide or announcement         |
+| spacebar (when in a shabad)                | Switch between main line and unchecked line |
+| spacebar (when a waheguru slide is active) | switch back to shabad                       |
```
Comment (tsingh777) [inline LEGEND.md]: @maneetpaul can you approve the copy for the text? should probably keep capitalization consistent

Comment (maneetpaul) [reply LEGEND.md]: Yes - need to capitalize the "s" in the last switch. And also all the ctrl and cmds should be caps CTRL CMD

File: LEGEND.md [hunk 2]
```diff
@@ -26,47 +26,47 @@
...
+| CMD + 1                                    | Waheguru Slide                              |
+| CMD + 2                                    | Mool Mantra Slide                           |
+| CMD + 3                                    | Empty Slide                                 |
+| CMD + 4                                    | Anand Sahib (Bhog)                          |
+| CMD + 5                                    | Help Guide                                  |
+| CMD + 6                                    | Shortcut Legend                             |
+| CMD + /                                    | Jump to search bar                          |
+| CMD + g                                    | Add Dhan Guru slide or announcement         |
+| Spacebar (when in a shabad)                | Switch between main line and unchecked line |
+| Spacebar (when a special slide [ie waheguru slide, announcement, etc) is active] | Switch back to shabad                       |
```
Comment (maneetpaul) [inline LEGEND.md]: Sorry to nit pick - the ()[] are incorrect in this one

File: LEGEND.md [hunk 3]
```diff
@@ -26,47 +26,47 @@
...
 #### <a name ="windows-others">Windows:</a>
 
 | Combination | Description       |
 | ----------- | ----------------- |
-| ctrl + q    | close app         |
-| ctrl + u    | check for updates |
-| ctrl + m    | minimize window   |
+| CTRL + q    | close app         |
+| CTRL + u    | check for updates |
+| CTRL + m    | minimize window   |
```
Comment (maneetpaul) [inline LEGEND.md]: Can we capitalize all the individual letters as well as the descriptions

**Pull request merged**